starfish/core/spots/DecodeSpots/trace_builders.py

```diff
@@ -0,0 +1,26 @@
+from starfish.core.intensity_table.intensity_table import IntensityTable
+from starfish.core.types import Axes, Features, SpotFindingResults
+
+
+def build_spot_traces_exact_match(spot_results: SpotFindingResults) -> IntensityTable:
+    """
+    Combines spots found in matching x/y positions across rounds and channels of
+    an ImageStack into traces represented as an IntensityTable.
+
+    Parameters
+    -----------
+    spot_results: SpotFindingResults
+        Spots found accrss rounds/channels of an ImageStack
+    """
+    # create IntensityTable with same x/y/z info accross all r/ch
+    spot_attributes = list(spot_results.values())[0]
+    ch_labels, round_labels = spot_results.ch_labels, spot_results.round_labels
+    intensity_table = IntensityTable.zeros(
+        spot_attributes=spot_attributes,
+        round_labels=round_labels,
+        ch_labels=ch_labels,
+    )
+    for r, c in spot_results.keys():
+        intensity_table.loc[dict(c=c, r=r)] = spot_results[{
```

Comment:
Meta question: I wonder if this can be done with more parallelism.

Reply:
potentially...if you first merge all the spotattribute tables..this might be some pandas sorcery I need @dganguli 's help on

Reply:
Doesn't need to happen now.